spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -154,3 +154,9 @@ For instance the following config will disable bulkhead policy globally:
 Policy will be disabled everywhere ignoring existing policy annotations on methods and classes.
 
 If the above configurations patterns are used with another value than `false` (i.e. `<classname>/<methodname>/<annotation>/enabled=whatever`) non-portable behaviour results.
+
+=== Configuring Metrics Integration
+
+The integration with MicroProfile Metrics can be disabled by setting a config property named `MP_Fault_Tolerance_Metrics_Enabled` to the value `false`.
```

Comment:
Since MP configuration values can be changed during runtime it should be specified what happens if the value changes. I guess any change during runtime wouldn't have any effect but it's worth to clarify this.

Reply:
Good point, I will add this.

Reply:
Done